lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -1178,36 +1177,44 @@ impl display::Object for Scene {
 }
 
 
+
+// ===============
+// === DomPath ===
+// ===============
+
+/// Abstraction for DOM path. It can be either a specific HTML element or a string which will be 
+/// used to look up for the element by its id.
```

Comment:
I don't quite get, what "string which be used to lookup by ID" means. Does it mean that the string is an ID? Or maybe it's a sort of pattern?

Reply:
I changed it to "DOM id", should be better now :)